.travis.yml

```diff
@@ -1,36 +1,55 @@
 language: python
 
-sudo: false
```

Comment:
sudo false had to go:
1. travis by default uses false
2. osx have no sudo, and it would fail on osx